packages/block-editor/src/store/reducer.js

```diff
@@ -637,70 +594,17 @@ const withInnerBlocksRemoveCascade = ( reducer ) => ( state, action ) => {
  */
 const withBlockReset = ( reducer ) => ( state, action ) => {
```

Comment:
We can probably now remove these HoR and just move its logic to the sub reducers.